Update __init__.py

afmklk opened the pull request:
Replaced deprecated call to hass.config_entries.async_forward_entry_setup with hass.config_entries.async_forward_entry_setup**s** to ensure compatibility with Home Assistant 2025.6, see https://developers.home-assistant.io/blog/2024/06/12/async_forward_entry_setups/

Have not tried this on a running system, so may need debugging.

afmklk committed:
Update __init__.py

Replaced deprecated async_forward_entry_setup to async_forward_entry_setups to ensure compatibility with Home Assistant 2025.6.

everdrift commented:
Did the same, it only prevents an error in the logs, blindscontrol integration still not working

afmklk committed:
Update __init__.py

Updated fix as suggested by @tcernik https://github.com/MatthewOnTour/BUT_blinds_time_control/issues/10#issuecomment-2994444157

afmklk commented:
See this fork for a working alternative: https://github.com/YanBad/BUT_blinds_time_control
